### PR TITLE
Knative Test Cases Rearrangement

### PR DESCRIPTION
**Pull request by gajanan-more**
Fixes: https://issues.redhat.com/browse/ODC-5073

Problem: Gherkin Scripts Rearrangement for Knative

Solution: We have created multiple feature files as per 4.6 epics for Knative. Let's rearrange and make the repo clean.

### CODE REVIEW

File: frontend/packages/dev-console/integration-tests/features/Knative/actions-on-knative-service.feature [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+   As a user I want to perform edit or delete operations and Set Traffic Distribution on knative Service in topology page
+
+Background:
+   Given user has installed OpenShift Serverless Operator
+   And user is at developer perspective
+   And user has selected namespace "aut-create-knative-actions-service"
+   And user has created knative service "nodejs-ex-git"
+
+
+@regression-1
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/Knative/actions-on-knative-service.feature:11]: @gajanan-more - I added -1 for debugging purpose, Can you please them from all the files

File: frontend/packages/dev-console/integration-tests/features/Knative/actions-on-knative-service.feature [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+   And user clicks the save button on the "Edit Application Grouping" modal
+   And user searches for application name "openshift-app"
+   And user clicks on "openshift-app" on topology page
+   Then updated service is present in side bar
+
+@debug
+Scenario: Perform cancel operation while editing application group : Kn-04-TC08
+   Given user is at the Topology page
+   When user selects "Edit Application Grouping" context menu option of knative service "nodejs-ex-git"
+   And user selects the "openshift-app" option from "application" drop down present in "Edit Application Grouping" modal
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/Knative/actions-on-knative-service.feature:121]:
```suggestion
  And user selects the "openshift-app" option from application drop down present in "Edit Application Grouping" modal
```

File: frontend/packages/dev-console/integration-tests/features/Knative/actions-on-knative-service.feature [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+   Then message should display as "{service name} has been updated to version {nnnnnn}"
+   And another message should display as "This object has been updated."
+
+
+@regression
+Scenario: Update the service to different application group existing in same project : Kn-04-TC07
+   Given user has created a workload named "openshift-app"
+   And user is at the Topology page
+   When user selects "Edit Application Grouping" context menu option of knative service "nodejs-ex-git"
+   And user selects the "openshift-app" option from "application" drop down present in "Edit Application Grouping" modal
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/Knative/actions-on-knative-service.feature:111]:
```suggestion
   And user selects the "openshift-app" option from application drop down present in "Edit Application Grouping" modal
```

File: frontend/packages/dev-console/integration-tests/features/Knative/actions-on-knative-service.feature [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+   And user searches for application name "openshift-app"
+   And user clicks on "openshift-app" on topology page
+   Then updated service should not display in side bar
+
+
+@regression
+Scenario: Update the service to new application group : Kn-04-TC06 
+   Given user is at the Topology page
+   When user selects "Edit Application Grouping" context menu option of knative service "nodejs-ex-git"
+   And user selects the "Create Application" option from "application" drop down present in "Edit Application Grouping" modal
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/Knative/actions-on-knative-service.feature:132]:
```suggestion
   And user selects the "openshift-app" option from application drop down present in "Edit Application Grouping" modal
```

**Review by makambalaji — CHANGES_REQUESTED** [review]
Please remove @debug and modify the @regression-1 to @regression where ever applicable in this entire pr

File: frontend/packages/dev-console/integration-tests/features/Knative/actions-on-knative-service.feature (hunk 1 above)
Comment (gajanan-more) [reply frontend/packages/dev-console/integration-tests/features/Knative/actions-on-knative-service.feature:11]: Updated.

File: frontend/packages/dev-console/integration-tests/features/Knative/actions-on-knative-service.feature [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+   Given user is at the Topology page
+   And number of annotations are "5" present in "nodejs-ex-git" service side bar details tab
+   When user selects "Edit Annotations" context menu option of knative service "nodejs-ex-git"
+   And user clicks Add button on the Edit Annotations modal
+   And user enters annotation key as "serving.knative.qe/creator "
+   And user enters annotation value as "kube:admin"
+   And user clicks the save button on the "Edit annotations" modal
+   Then number of Annotations increased to "6" in "nodejs-ex-git" service side bar details
+
+@debug
```
[the remark this answers is not in the document]
Comment (gajanan-more) [reply frontend/packages/dev-console/integration-tests/features/Knative/actions-on-knative-service.feature:76]: Updated.

File: frontend/packages/dev-console/integration-tests/features/Knative/actions-on-knative-service.feature (hunk 2 above)
Comment (gajanan-more) [reply frontend/packages/dev-console/integration-tests/features/Knative/actions-on-knative-service.feature:121]: Updated.

File: frontend/packages/dev-console/integration-tests/features/Knative/actions-on-knative-service.feature (hunk 3 above)
Comment (gajanan-more) [reply frontend/packages/dev-console/integration-tests/features/Knative/actions-on-knative-service.feature:111]: Updated.

File: frontend/packages/dev-console/integration-tests/features/Knative/actions-on-knative-service.feature (hunk 4 above)
Comment (gajanan-more) [reply frontend/packages/dev-console/integration-tests/features/Knative/actions-on-knative-service.feature:132]: Updated.

**Pull request merged**